docs/list-available-operators.md

```diff
@@ -0,0 +1,31 @@
+# How do I list Operators available to install?
+
+There is an `APIService` in OLM named `PackageManifest` that contains information existing `CatalogSources`, which is essentially a repository of CSVs, CRDs, and packages that define an application, and their `ConfigMaps` in the cluster. By querying that API, you can see the list of available operators.
+
+There are two different types of `CatalogSource` in OLM: global and namespace `CatalogSource`. The global `CatalogSource` contains operators that will be available for all namespaces while namespace `CatalogSource` only contains operators that are only available for a specific namespace.
```

Comment:
nit: `namespaced`

Reply:
Was there a reason this wasn't changed to `namespaced` @dinhxuanvu?